### PR TITLE
Phone Component

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    const parsedNumber = parseNumber(number, defaultCountry)
+
+    if (onChange) {
+      const numberDB = _isEmpty(parsedNumber) ? removeSpaces(number) : formatNumber(parsedNumber, 'E.164')
+      onChange(numberDB)
+    }
+
+    const currentCaretPosition = e.target.selectionStart
+    repositionCaret(number, currentCaretPosition)
+  }
```
Comment (rzueger) [inline packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.js]: do we really have to define the functions above inside the render function?

File: packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import _isEmpty from 'lodash/isEmpty'
+import {AsYouType, formatNumber, parseNumber} from 'libphonenumber-js'
+
+const DEFAULT_DEFAULT_COUNTRY = 'CH'
```
Comment (rzueger) [inline packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.js]: :)

File: packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    }
+  }
+
+  const handleChange = e => {
+    const number = e.target.value
+    const parsedNumber = parseNumber(number, defaultCountry)
+
+    if (onChange) {
+      const numberDB = _isEmpty(parsedNumber) ? removeSpaces(number) : formatNumber(parsedNumber, 'E.164')
+      onChange(numberDB)
```
Comment (rzueger) [inline packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.js]: what does `numberDB` stand for?

File: packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PhoneFormatter from './PhoneFormatter'
+import {shallow} from 'enzyme'
+
+describe('tocco-ui', () => {
+  describe('FormattedValue', () => {
```
Comment (rzueger) [inline packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js]: no `typeFormatters` package in between?

Comment (MethodenMann) [reply packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js]: added to all tests

**Pull request merged**